agent/bench-scripts/pbench-netperf

```diff
@@ -532,10 +526,10 @@ for protocol in `echo $protocols | sed -e s/,/" "/g`; do
 								fi
 								pbench-stop-tools --group=$tool_group --dir=$benchmark_results_dir
 								pbench-send-tools --group=$tool_group --dir=$benchmark_results_dir
-								pbench-postprocess-tools --group=$tool_group --dir=$benchmark_results_dir
 							else
 								echo "Not going to run netperf.  Only postprocesing existing data"
 							fi
+							pbench-postprocess-tools --group=$tool_group --dir=$benchmark_results_dir
```

Comment:
This is a bug fix to allow post-processing of tool data with `--postprocess_only=y`.